app/employee-app/backend/modules/types/constants.bal

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2024 WSO2 LLC. (http://www.wso2.org).
```

Comment:
Shouldn't the types module be renamed to something like commons since we also have constants and enums in this module.

Reply:
As discussed in the meeting, we've removed types module and included necessary files inside the root directory.